templates/taxi/car_detail.html

```diff
@@ -1,22 +1,33 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <h1>Model: {{ car.model }}</h1>
+  <p><b>Manufacturer: </b>({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <div>
+    {% if user in car.drivers.all %}
```

Comment:
The condition `user in car.drivers.all` should use parentheses to call the `all()` method, i.e., `user in car.drivers.all()`. This ensures the method is executed and returns the correct queryset.